pkg/envtest/crd.go

```diff
@@ -201,7 +201,7 @@ func readCRDs(path string) ([]*apiextensionsv1beta1.CustomResourceDefinition, er
 		}
 
 		// Unmarshal the file into a struct
-		b, err := ioutil.ReadFile(filepath.Join(path, file.Name()))
+		b, err := ioutil.ReadFile(filepath.Join(path, file.Name())) // nolint: gosec
```

Comment:
Can we configure the gosec linter to ignore these types or errors (or specific files) ? Annotating the source code to exclude is not right.